fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/session/HttpSession.java

```diff
@@ -27,32 +27,25 @@
 
```

Comment:
How will the interaction between HttpSession and FedoraTransaction be managed?
- Commit on HttpSession will commit FedoraTransaction? (Short-Lived vs Long)
- Expire on HttpSession will delete (or mark expired) the FedoraTransaction? (Short-Lived vs Long)

Where will the rollback of the FedoraTransaction be triggered if the services encounter rollback in interaction with persistence layer?

Reply:
That is a good question.  I believe the answer is transaction expiration will result in a rollback.   So if a transaction is expired when the request comes in it would be rolled back.  (I think).   I'm curious what others think.

Reply:
Decision from meeting: HttpSession can be deleted.

Reply:
Is this class being deleted or is that happening later?